### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by kariabancroft**
Here are my preliminary changes per my discussion w/ @jnf yesterday.

I would like to spend some time thinking about/discussing the goals of the capstone and solidifying those more in this documentation. 
cc: @sudocrystal

### CODE REVIEW

File: topic_resources/capstone/presentation.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
+The capstone presentation highlights your application as the culmination of your work at Ada.
+It should be __no more than 5 minutes__ in length and should have the following components: 
 
 - What problem does the app solve? Who does it solve it for?
 - What are 1-2 major features of the application, demoed from the eyes of a user?
-- What was one technical challenge you overcame? 
+- What was one technical challenge you overcame?
 
 Prior to demoing, you should have the following:
 - Provide URLs to the first person presenting in your group; ensure you have credentials for accounts accessible
```
Comment (jnf) [inline topic_resources/capstone/presentation.md]: We ended up not doing this at demos for C3. Everyone showed from their own laptop. It was moderately obnoxious, but with proper prep works just fine. Dunno if we wanna do it that way again or not. One thing I'd add would be a requirement for folks with hardware or props or whatevs to clear the space before the next person goes on.

File: cohort_schedules/C04_classroom.md [hunk 2]
```diff
@@ -336,7 +336,7 @@ Week 11 brings the first group, multi-week project for students. This is a major
 In addition to all of this, students are learning how to perform user authentication in Rails using session data and controller filters. Advanced validations for model classes are also discussed, and the bEtsy project provides several opportunities for using custom validations.
 
 ### Learning Goals
-- Long-term project development with the Agile methodology 
+- Long-term project development with the Agile methodology
 - Different team member roles (task manager, stand up leader, project manager)
 - Password-based user authentication in Rails
 - Authorization of specific actions/routes in Rails
```
Comment (sudocrystal) [inline cohort_schedules/C04_classroom.md]: This was something that was brought up in feedback: not being clear about what the goals of the capstone were/are. So just a note to us to remember to go over these goals with the students at the beginning (and then revisiting a couple times during capstone).

Also making sure that these are the goals that we really want.

File: topic_resources/capstone/capstone.md [hunk 3]
```diff
@@ -1,51 +1,27 @@
...
-The Capstone Project is the largest undertaking you'll experience in the Ada classroom.
+### Teams
+For the capstone project, you have the option to work individually or in a pair. If you do decide to work individually, you will have another individual with whom you will work with throughout the capstone as a sounding board and support system even though you will be working on different projects.
 
-Not only is it a month long, but you will be your own product owner. It will be up to you to decide (with input from instructors, mentors, and TAs) what gets built when (and how).
+### Goals
+- Demonstrate self-direction, time management, and independent learning
+- Learn and implement new technologies
+- Complete a product lifecycle from conception to delivery
+- Utilize agile practices learned to assist in project completion
```
Comment (sudocrystal) [inline topic_resources/capstone/capstone.md]: This was something that was brought up in feedback: not being clear about what the goals of the capstone were/are. So just a note to us to remember to go over these goals with the students at the beginning (and then revisiting a couple times during capstone).

Also making sure that these are the goals that we really want.

File: topic_resources/capstone/features.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Capstone Technology Features
+One of the most exciting and challenging things about the capstone project is learning new technologies to integrate into your project.
```
Comment (sudocrystal) [inline topic_resources/capstone/features.md]: Where is this page linked on the main capstone page?

**Pull request merged**